main.tf

```diff
@@ -407,3 +407,41 @@ resource "github_repository_project" "repository_project" {
   name       = each.value.name
   body       = each.value.body
 }
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Webhooks
+# ---------------------------------------------------------------------------------------------------------------------
+locals {
+  webhooks = [
+    for b in var.webhooks : merge({
+      name   = null
+      active = null
+      events = []
+      configuration = try(
+        merge({
+          url          = null
+          content_type = null
+          insecure_ssl = null
+          secret       = null
+        }, b.configuration)
+        , {}
+      )
+    }, b)
+  ]
```

Comment:
we saw structures like this failing in the past. This was caused by HCl internals of doing deep compare in list of objects. stating every object needs to be the same. 

but as tests pass I would say, let's care when it fails ;)